liferay-frontend/liferay-portal
# LPS-150360 CSS fixes & improvements for FDS

[Jira ticket](https://issues.liferay.com/browse/LPS-150360)

## Motivation
As reflected in [this document](https://liferay.atlassian.net/wiki/spaces/ENGFRONTENDINFRA/pages/2002681985/FDS+technical+improvements+checklist#Styles), the FDS styles have no scope and they bleed out to the page. Also, some reorganisation can be beneficial in the future.

## Proposed solution
- All files are now grouped
- Most of the files are scoped inside a `.fds` class
- Global files (due to the use of React Portal, global tooltips, etc..) have been modified to apply a more specific class, ie: use `fds-side-panel` instead of `side-panel` which eventually can conflict with globally declared classes in Clay or other modules. Also, global CSS files have been reorganised for clarity.
- Some CSS code has been deleted as no usages have been found, nor in the portal or Clay.
- Existent CSS for data renderers are now under the same sub-folder

## How to test this PR
NOTE: This is the most delicate point of this PR. I've tested everything I could but my lack of knowledge regarding the usages of the FDS makes it so hard to check every class.

- Deploy FDS sample module, place it in a page, check the styles are normal
- Create a new site with Minion template
  - Navigate to Commerce > Inventory, check the styles are normal
  - Open a Product, check the styles are normal
  - Open a Wharehouse, Side Panel will appear, check the styles are normal
- Navigate to Object > PIcklist
  - Create a picklist and navigate to its detail, Side Panel will appear, check the styles are normal
  - Add an item to the picklist, a modal will appear, check the styles are normal
  - Navigate to the picklist's item, a modal will appear, check the styles are normal
